source/nodejs/adaptivecards-fabric/src/components/inputs/InputTextFabric.tsx

```diff
@@ -20,20 +20,29 @@ export class InputTextFabric extends Shared.ReactInputElement {
     public getJsonTypeName = (): string => "Input.Text";
 
     public parse = (json: any, errors?: any) => {
-        this.valueInternal = json.value;
```

Comment:
You don't appear to be parsing/handling the `inlineAction` and `style` properties.

Reply:
inlineAction is not supported by this version. adding style

Reply:
Not sure what you mean. inlineAction is definitely supported in 1.2, which has shipped. inlineAction isn't a native capability of the input control, it needs to be handled explicitly.